docs/sources/admin/updating.md

```diff
@@ -1 +1,23 @@
 # Updating the Tool
+
+## User and Role Management
+
+Access to the matching tool web application is handled with users and roles.
+
+A **user** is anybody who can log into the tool.
+
+A **role** is a specific function of the tool that can be accessed by users. In the matching tool, this is the combination of two things: A `jurisdiction` (e.g. your county) and `event_type` (e.g. hmis_service_stays).
+
+The matching-tool repository comes with a convenience script for adding users and roles that aims to make the usual use case (one jurisdiction, with users who have access to many roles) easier. This was run as part of the [install docs](install.md), and you can use it later to add more users.
+
+It is located in the repository path `/scripts/3_create_user`. To run it, you need the following information:
+
+- A short, lowercase name for your jurisdiction (e.g. 'cook'). Don't include spaces as it will confuse the script. This is just the name that shows up at the top of the webapp. This should match whatever jurisdiction name that was picked on initial tool installation. If you don't know what this is, you can query the `public.role` table in the webapp database to see what role names exist.
```

Comment:
>  This is just the name that shows up at the top of the webapp.

This sounds like it is what is displayed to the user in the browser. Is that correct?